internal/test-utils/test_utils.go

```diff
@@ -19,3 +19,10 @@ func ExpectResourceNotExists(resource client.Object, request common.Request) {
 	ExpectWithOffset(1, err).To(HaveOccurred())
 	ExpectWithOffset(1, errors.IsNotFound(err)).To(BeTrue())
 }
+
+func ExpectNamespacedResourceExists(resource client.Object, request common.Request, namespace string) {
+	key := client.ObjectKeyFromObject(resource)
+	err := request.Client.Get(request.Context, key, resource)
+	ExpectWithOffset(1, err).ToNot(HaveOccurred())
```

Comment:
IMO either use `...WithOffset` in all assertions or in none. I'd vote for using the standard `Expect`.

Reply:
+1 for the consistency. 
But in this case using no offset will generate useless error message, because it will point to this line without any information about the calling function or the values of parameters passed.